### PR TITLE
Included support for RHEL 7.2

### PR DESCRIPTION
**Pull request by sushilrai**
- Added support for installing additional packages / group packages
- Cleanup of non-installation so that they can be used as storage disks for HCI use cases

### CODE REVIEW

File: data/templates/centos-ks [hunk 1]
```diff
@@ -147,6 +156,15 @@ export PATH
     <% } %>
 <% } %>
 
+# Cleanup non-installation disks
+<% if (typeof clearDisk !== 'undefined' ) { %>
```
Comment (yyscamper) [inline data/templates/centos-ks]: the `clearDisk` is boolean, it need be `true` at here.

**Comment by RackHDCI** [comment]
The Copyright Check has failed!
The files listed below do not have a valid copyright statement:
- data/templates/centos-ks

**Comment by nortonluo** [comment]
The CI test requires copy rights line in the code.  eg."Copyright 2017, DELL, Inc." Otherwise it will failed.

It use PackagesArray in the https://github.com/RackHD/on-tasks/pull/568, and it use key word package here. Any relations?

**Comment by sushilrai** [comment]
@RackHDCI I have aligned one more PR based on this PR. Please suggest if any more changes are required for these changes to be merged in master branch.

**Pull request merged**